models/db/engine.go

```diff
@@ -160,9 +160,9 @@ func NewInstallTestEngine(ctx context.Context, migrateFunc func(*xorm.Engine) er
 	return syncTables()
 }
 
-// SetEngine sets the xorm.Engine
-func SetEngine() (err error) {
-	x, err = GetNewEngine()
+// InitEngine sets the xorm.Engine
+func InitEngine() (err error) {
+	x, err = NewEngine()
```

Comment:
I know it is only a refactoring, but should we shortcut when x is already set?
What I mean is:
```suggestion
	if x != nil {
		return nil
	}
	x, err = NewEngine()
```

Reply:
I think it may breaks something if we check that. I will left it other PRs. That means should we allow reinitialize the engine?

Reply:
To me at least, it looks like we allow that at the moment…